CHANGELOG.md

```diff
@@ -3,6 +3,17 @@
 All notable changes to this project will be documented in this file.
 This project adheres to [Semantic Versioning](http://semver.org/).
 
+## [6.5.1]
+
+### Added
+
+- Privacy manifest file which correctly declares how and which user data is tracked [source](https://developer.apple.com/videos/play/wwdc2023/10060/?time=139) 
```

Comment:
SDK now includes a new privacy policy file indicating SDK's usage of user data. 

The link is not required.